rebar.config

```diff
@@ -1,17 +1,36 @@
-{deps, []}.
+{cover_export_enabled, true}.
+
+{cover_enabled, true}.
+
+{deps, [
+    {erlang_lorawan, {git, "https://github.com/helium/erlang-lorawan.git"}},
+    {grpc_client, {git, "https://github.com/Bluehouse-Technology/grpc_client.git"}},
+    {grpcbox, {git, "https://github.com/novalabsxyz/grpcbox.git"}},
+    {helium_proto, {git, "https://github.com/helium/proto.git"}},
+    {lager, "3.9.2"}
+]}.
 
 {erl_opts, [
     debug_info,
+    {parse_transform, lager_transform},
     warnings_as_errors
 ]}.
 
+{grpc, [
```

Comment:
i dont think you need this section

Reply:
Removed with commit 2181801

Confirmed: `make clean ; rebar3 grpc gen` still works